### PR TITLE
Adds scale prop to radio group, updates style

### PR DESCRIPTION
**Pull request by macandcheese**
- Updates styles a bit to match those from recent designs.
- Adds scale prop (s/m/l)
- Adds prop validation for theme, scale
- Prettifies files
- Updates preview bg color of index.html

### CODE REVIEW

**Comment by paulcpederson** [comment]
Looks good, you will probably need to pull from master and resolve the `CHANGELOG` conflict

**Comment by macandcheese** [comment]
@paulcpederson merged in master 👍

**Comment by paulcpederson** [comment]
Going to merge this in, we can make updates later if @jcfranco reviews

File: src/components/calcite-radio-group/calcite-radio-group.tsx [hunk 1]
```diff
@@ -102,6 +99,14 @@ export class CalciteRadioGroup {
   //--------------------------------------------------------------------------
 
   connectedCallback() {
+
+    // prop validations
+    let scale = ["s", "m", "l"];
+    if (!scale.includes(this.scale)) this.scale = "m";
```
Comment (jcfranco) [inline src/components/calcite-radio-group/calcite-radio-group.tsx]: Can you add a test for this?

**Pull request merged**